.evergreen/connect.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
The motivation for moving this into a separate script file is to make it easier to run locally. I build in a directory named `cmake-build` and can run this locally as follows:

```
MONGOC_INSTALL_PREFIX=/Users/kevin.albertson/install/mongo-c-driver-1.18.0 \
MONGOCXX_INSTALL_PREFIX=/Users/kevin.albertson/install/mongo-cxx-driver-dev \
LIB_DIR="lib" \
BUILD_TYPE="Debug" \
BUILD_DIR=$(pwd)/cmake-build \
URI="mongodb://localhost:27017/?" \
    ./.evergreen/connect.sh
```

Reply:
Is there a place (even as an extended comment at the beginning of the script, but perhaps as a help screen?) where you can put a scrubbed version of this example? Sounds very useful!

Reply:
Good idea, done!